starter/slack-spring-boot/src/main/kotlin/com/kreait/slack/broker/exception/SlackExceptionHandler.kt

```diff
@@ -30,6 +30,24 @@ class SlackExceptionHandler(private val errorResponse: String) {
         return ResponseEntity.ok(DialogErrorResponse(ex.errors))
     }
 
+    /**
+     * Handle [IllegalArgumentException]s
+     * This will also handle wrapped [DialogValidationException]s and return the correct type
```

Comment:
can you mention that we unwrap IllegalArgument here to see if it is a wrapped  jackson exception (since jackson wraps it)